smartdashboard/view_builders.py

```diff
@@ -466,3 +445,151 @@ def overview_builder(manifest: Manifest) -> OverviewView:
         ens_view = ens_builder(manifest)
 
     return OverviewView(exp_view, app_view, orc_view, ens_view)
+
+
+def db_telem_builder(manifest: Manifest) -> TelemetryView:
+    """Database Telemetry page to be rendered
+
+    This function organizes the views within
+    the Database Telemetry page.
+
+    :param manifest: Manifest of the Experiment
+    :type manifest: Manifest
+    :return: View of the DB Telemetry Page
+    :rtype: TelemetryView
+    """
+    st.header("Database Telemetry")
+
+    st.write("")
+
+    col1, _ = st.columns([6, 6])
+    with col1:
+        selected_orchestrator_tuple = st.selectbox(
+            "Select an orchestrator:",
+            manifest.orcs_with_run_ctx,
+            format_func=lambda tup: f"{tup[1].name}: Run {tup[0]}",
```

Comment:
Done! Couldn't do a `NamedTuple` because it doesn't allow multiple inheritance so i did a frozen, generic dataclass instead. Lemme know what you think!

Reply:
Doh! I forgot that was a 3.11 feature. Yeah, I like the dataclass!!